Tasks/Src/Task_VoltTempMonitor.c

```diff
@@ -112,8 +115,19 @@ void Task_VoltTempMonitor(void *p_arg) {
             assertOSError(err);
 
             temperatureHasBeenChecked = true;
+        } 
+        //PID loop - sets fan speed based on avg temperature and desired temperature
+        //overrides PID loop if above PID_MAX_TEMPERATURE or if it's FAULT
+        if (PIDStatus != SAFE || temperatureStatus != SAFE) {    
```

Comment:
if temperatureStatus != SAFE, then this code will never run. It would just make more sense to do something like this at line 103.
`if (Temperature_General_Check(PID_MAX_TEMPERATURE) != SAFE) BSP_Fans_SetAll(TOPSPEED);`
and so on. Also, it is kind of weird to have both functions (Temperature_CheckStatus and the one above) do basically the same thing with a slight change. I would add a TODO saying to merge the 2 functions together later down the line. For now it is fine though.